privacyidea/static/components/token/views/token.enroll.hotp.html

```diff
@@ -1,10 +1,30 @@
-<p class="help-block"
-   ng-hide="$state.includes('token.wizard')"
-   translate>
-The HOTP token is an event based token. You can paste a secret key or
-    have the server generate the secret and scan the QR code with a smartphone
-    app like the Google Authenticator turning your
-    smartphone into an authentication device.
+<p>
```

Comment:
Can we make some kind of angular component (or whatever this is called) from this part so we don't have to edit all three parts?

Reply:
No I would not want to do this.

Reply:
ok